Orange/classification/rules.py

```diff
@@ -1,3 +1,12 @@
+"""
+Induction of rules works by finding a rule that covers some learning instances,
+removing these instances, and repeating this until all instances are covered.
+The rules are scored by heuristics such as impurity of class distribution of
+the covered instances. The module includes the common rule-learning algorithms,
```

Comment:
Since you're not mentioning what these two algorithms are, I reckon no. I'm no expert, mind you, but I'd go with either *two common rule-learning algorithms* or without *the*. Especially since it's plural. My 2c.